src/data/representative_content.py

```diff
@@ -0,0 +1,45 @@
+# Gets 1000 most 'representative' pieces of content for each top level taxon and it's children
+# Saves them to file
+
+import os
+import pandas as pd
+from sklearn.metrics import pairwise_distances_chunked
+from utils import directories as dirs
+
+class RepresentativeContent:
+    def __init__(self, content, tree):
+        self.content = content
+        self.tree = tree
+
+    def generate(self):
+        well_placed_content = {}
+        for apex_node in self.tree.apex_nodes():
+            well_placed_content[apex_node.title] = self.find_well_placed_items_in_apex_node(apex_node)
+        return well_placed_content
+
+    # Finds all content that it considers to be representative of a branch node
+    # and all it's children
+    def find_well_placed_items_in_apex_node(self, apex_node):
```

Comment:
Some description of the return params might be in order here, to make it more obvious that the return value of this function is actually a CSV file path rather than whatever a well placed item might otherwise be and/or consider creating a new function from everything after the _return_ to end of method that better encapsulate that particular logic.